### DISCUSSION TITLE
Add Orientation / Frequency to the Triggers UI

### DISCUSSION THREAD

**Discussion by MartinSWDev** [supabase/supabase#27832] [Feature Requests] [answered]
Currently when creating a trigger you can select either statement or row for the "orientation" / "frequency" (should probably stick to one word ? ) but after creation this is not visible in the UI

In addition, the default is statement, could we consider swapping this to row, as "new" is null on statements and creates non obvious errors as to why its null

<img width="1" alt="Screenshot 2024-07-05 at 22 51 27" src="https://github.com/supabase/supabase/assets/98239525/5747cf46-95d9-4430-82b6-aa6a573030da">
<img width="1409" alt="Screenshot 2024-07-05 at 22 51 34" src="https://github.com/supabase/supabase/assets/98239525/29e6118b-3952-4f90-b9ee-3d7ba3dc9c03">
<img width="393" alt="Screenshot 2024-07-05 at 22 52 05" src="https://github.com/supabase/supabase/assets/98239525/73a0d292-edd1-4ae0-a1ef-52c87c10bea0">
 in the UI

**Comment by GaryAustin1** [comment]
This is a great idea!  It hits a lot of users in the forums.

**Reply by kiwicopple** [reply, marked as answer]
this is now merged: https://x.com/martinswdev/status/1812991828968546322

**Comment by MartinSWDev** [comment]
Turn around on this was great! Thanks all :)
